servers/physics_2d/godot_shape_2d.cpp

```diff
@@ -371,7 +371,7 @@ void GodotCapsuleShape2D::get_supports(const Vector2 &p_normal, Vector2 *r_suppo
 	real_t d = n.y;
```

Comment:
This local `d` variable can be removed now, and its usage below replaced by `n.y`.